### DISCUSSION TITLE
ScriptReference/Camera.GetAllCameras

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#406] [Notes]
# ScriptReference/Camera.GetAllCameras

https://docs.unity3d.com/ScriptReference/Camera.GetAllCameras.html

**Comment by nindim** [comment]
This gives access to all the active cameras across all scenes including cameras inside DontDestroyOnLoad.  
The first line of the description would be more accurate as: "Fills an array of Camera with all the currently enabled cameras, without allocating a new array."
